python/tests/test_collections.py

```diff
@@ -0,0 +1,68 @@
+from unittest.mock import Mock
+
+import pytest
+from numerous import collection
+from numerous._client import Client
+from numerous.generated.graphql.client import Client as GQLCleint
+from numerous.generated.graphql.fragments import CollectionNotFound, CollectionReference
+
+
+ORGANIZATION_ID = "test_org"
+COLLECTION_NAME = "test_collection"
+NESTED_COLLECTION_NAME = "nested_test_collection"
+COLLECTION_REFERENCE = CollectionReference(key="test_key", id="test_id")
+NESTED_COLLECTION_REFERENCE=CollectionReference(key="nested_test_key",
```

Comment:
This is not well formatted. Try to run `ruff format` on the codebase. We should add a linter to check this as well, but I'll do that separately later :)